src/focus-visible.js

```diff
@@ -146,7 +146,10 @@ function init() {
       return;
     }
 
-    if (e.target.classList.contains('focus-visible')) {
+    if (
+      e.target.classList.contains('focus-visible') ||
+      el.hasAttribute('data-focus-visible-added')
```

Comment:
It doesn't look like `el` is in scope here. Did you mean `e.target`?

Reply:
I did indeed! Sorry copypasta... fixed.